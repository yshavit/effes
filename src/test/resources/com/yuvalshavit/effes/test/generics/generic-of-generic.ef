data Node[T](head: T, tail: Node[T] | Nothing)

data ListOfNodes[T](nodes: Node[T])

do some stuff with all that
