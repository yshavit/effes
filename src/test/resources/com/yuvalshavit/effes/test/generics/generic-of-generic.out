ListOfNodes(Node(True, Node(False, Node(True, Nothing))))
>> null
